api/v1alpha1/worker_types.go

```diff
@@ -122,6 +122,9 @@ type TemporalWorkerDeploymentStatus struct {
 
 	// TODO(jlegrone): Add description
 	VersionConflictToken []byte `json:"versionConflictToken"`
+
+	// ExternallyModified indicates if the worker deployment was modified outside of the controller (e.g. via CLI)
+	ExternallyModified bool `json:"externallyModified,omitempty"`
```

Comment:
Maybe it would be useful to humans if the `LastModifierIdentity` were encoded here as a string instead of a bool?

Reply:
Yeah I prefer that, keeps the `Status` as factual.